app/grandchallenge/notifications/models.py

```diff
@@ -354,28 +354,32 @@ def print_notification(self, user):  # noqa: C901
             self.type
             == NotificationType.NotificationTypeChoices.EVALUATION_STATUS
             and self.actor != user
-            and self.message == "failed"
+            and self.message != "succeeded"
         ):
+            if self.action_object.error_message:
+                error_message = format_html(
```

Comment:
Please use the `kwargs` style of `format_html`, I think it makes it easier to parse, e.g. https://github.com/comic/grand-challenge.org/blob/58ec92ab2b3b1f4aecaeebf672729600f0f8b421/app/grandchallenge/challenges/admin.py#L69-L76

Also update any usages you happen to touch.